### PR TITLE
PN_9153

### PR DESCRIPTION
**Pull request by azhansultan**
*Added missing steps and methods

### CODE REVIEW

File: src/test/resources/feature/3-destinatario/personaGiuridica/altriRecapitiPG/004_062_modificaPECPG.feature
Comment (leleOFA) [inline src/test/resources/feature/3-destinatario/personaGiuridica/altriRecapitiPG/004_062_modificaPECPG.feature]: modify deleting .yaml logic

File: src/main/java/it/pn/frontend/e2e/common/RecapitiDestinatarioPage.java
Comment (afus0) [inline src/main/java/it/pn/frontend/e2e/common/RecapitiDestinatarioPage.java:1]: I reworked this file as well: https://github.com/pagopa/pn-frontend-e2e/pull/130. 
Please try to merge the common changes

File: src/main/java/it/pn/frontend/e2e/common/RecapitiDestinatarioPage.java
Comment (afus0) [inline src/main/java/it/pn/frontend/e2e/common/RecapitiDestinatarioPage.java]: Check https://github.com/pagopa/pn-frontend-e2e/pull/130

File: src/main/java/it/pn/frontend/e2e/common/RecapitiDestinatarioPage.java
Comment (afus0) [inline src/main/java/it/pn/frontend/e2e/common/RecapitiDestinatarioPage.java:72]: Check https://github.com/pagopa/pn-frontend-e2e/pull/130

File: src/main/java/it/pn/frontend/e2e/common/RecapitiDestinatarioPage.java
Comment (afus0) [inline src/main/java/it/pn/frontend/e2e/common/RecapitiDestinatarioPage.java]: Check https://github.com/pagopa/pn-frontend-e2e/pull/130

File: src/main/java/it/pn/frontend/e2e/common/RecapitiDestinatarioPage.java
Comment (afus0) [inline src/main/java/it/pn/frontend/e2e/common/RecapitiDestinatarioPage.java]: Check https://github.com/pagopa/pn-frontend-e2e/pull/130

File: src/main/java/it/pn/frontend/e2e/common/RecapitiDestinatarioPage.java
Comment (afus0) [inline src/main/java/it/pn/frontend/e2e/common/RecapitiDestinatarioPage.java]: Check https://github.com/pagopa/pn-frontend-e2e/pull/130

File: src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaFisica/RecapitiPersonaFisicaTest.java
Comment (afus0) [inline src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaFisica/RecapitiPersonaFisicaTest.java]: Check https://github.com/pagopa/pn-frontend-e2e/pull/130

**Review by afus0 — CHANGES_REQUESTED** [review]
Most of the changes can be merged with this PR: https://github.com/pagopa/pn-frontend-e2e/pull/130. If not, please wait for approval from @leleOFA, and once approved, pull from the `develop` branch.

File: src/main/java/it/pn/frontend/e2e/common/RecapitiDestinatarioPage.java
Comment (afus0) [reply src/main/java/it/pn/frontend/e2e/common/RecapitiDestinatarioPage.java]: There is already the method above `clickSuModificaPEC().`

File: src/main/java/it/pn/frontend/e2e/common/RecapitiDestinatarioPage.java
Comment (afus0) [reply src/main/java/it/pn/frontend/e2e/common/RecapitiDestinatarioPage.java]: Take the id element or mix both with xpath and id

File: src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaFisica/RecapitiPersonaFisicaTest.java
Comment (afus0) [reply src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaFisica/RecapitiPersonaFisicaTest.java]: There is already a feature step:
``` 
@And("Nella pagina I Tuoi Recapiti si clicca sul bottone modifica PEC e si verifica che si possa modificare la PEC")`
@And("Si clicca sul bottone annulla per annullare la modifica della PEC e si verifica che non si possa modificare la PEC")
```

File: src/main/java/it/pn/frontend/e2e/common/RecapitiDestinatarioPage.java
Comment (azhansultan) [reply src/main/java/it/pn/frontend/e2e/common/RecapitiDestinatarioPage.java]: questo e click su Annulla

File: src/main/java/it/pn/frontend/e2e/common/RecapitiDestinatarioPage.java
Comment (azhansultan) [reply src/main/java/it/pn/frontend/e2e/common/RecapitiDestinatarioPage.java]: @ImJimMoriarty this element was not added by me, it is a space issue which I deleted. I can not change it now because it wil take a lot of time to replicate the issue and find the webelement

**Review by afus0 — APPROVED** [review]

**Pull request merged**
